resources/assets/uad_lists.json

```
@@ -30829,11 +30828,11 @@
   {
     "id": "com.google.android.overlay.gmsgsaconfig",
     "list": "Oem",
-    "description": "Overlay theme for gmsgsaconfig? Probably an RRO? https://source.android.com/devices/architecture/rros",
+    "description": "Not needed overlay, only has code to set googlequicksearchbox as default assistant",
```

```suggestion
    "description": "Not needed overlay, only has code to set `googlequicksearchbox` as default assistant",
```

---

Added App name to make it clarity.

```suggestion
    "description": "Not needed overlay, only has code to set `googlequicksearchbox`(Google App) as default assistant",
```